wiki/Chat_Console/th.md

```diff
@@ -1,102 +1,103 @@
-# หน้าต่างพูดคุยอย่างง่าย
+# หน้าต่างพูดคุย
 
-กด `F8` จากที่ไหนก็ได้ในเกม หรือกดปุ่ม `Show Chat` ที่อยู่มุมล่างขวาเพื่อแสดงหน้าต่างแชทที่มีขนาดหนึ่งในสามของหน้าจอ
+กด `F8` จากเกือบทุกที่ในเกม หรือกดปุ่ม `Show Chat` ที่อยู่มุมล่างขวา เพื่อแสดงหน้าต่างแชทที่มีขนาดหนึ่งในสามของหน้าจอ
 
-![Chat Console](img/Chatconsole1.png "Chat Console")
+![หน้าต่างแชท](img/Chatconsole1.png "หน้าต่างแชท")
 
-- แท็บที่เห็นจะแสดงช่องที่คุณสามารถเข้าไปดูได้ แค่กดที่แท็บก็เปลี่ยนช่องได้แล้ว หรืออาจจะกดปุ่ม `+` สีเหลืองเพื่อเลือกช่องใหม่ที่ต้องการ
-- สีของผู้เล่นต่างๆ มีความหมายต่างกัน
+- แท็บที่เห็นจะแสดงช่องที่คุณสามารถเข้าไปดูได้ แค่กดที่แท็บก็เปลี่ยนช่องได้แล้ว กดปุ่ม `New` เพื่อเปิดรายชื่อช่องสำหรับเข้าร่วม
```

Comment:
```suggestion
- แท็บที่เห็นจะแสดงช่องที่คุณสามารถเข้าไปดูได้ แค่กดที่แท็บก็เปลี่ยนช่องได้แล้ว กดปุ่มบวกสีเหลืองเพื่อเปิดรายชื่อช่องสำหรับเข้าร่วม
```
Change because of #4435